cli/flags.rs

```diff
@@ -171,62 +170,75 @@ pub fn set_flags(
         .long("prefetch")
         .help("Prefetch the dependencies"),
     ).subcommand(
-      // TODO(bartlomieju): version is not handled properly
       SubCommand::with_name("info")
+        .setting(AppSettings::DisableVersion)
         .about("Show source file related info")
         .arg(Arg::with_name("file").takes_value(true).required(true)),
     ).subcommand(
-      // TODO(bartlomieju): version is not handled properly
-      SubCommand::with_name("fmt").about("Format files").arg(
-        Arg::with_name("files")
-          .takes_value(true)
-          .multiple(true)
-          .required(true),
-      ),
+      SubCommand::with_name("eval")
+        .setting(AppSettings::DisableVersion)
+        .about("Eval script")
+        .arg(Arg::with_name("script").takes_value(true).required(true)),
```

Comment:
maybe call this "code" instead of "script" ?

Reply:
Changed to `code`